homeassistant/components/facebox/image_processing.py

```diff
@@ -47,6 +48,7 @@
         vol.Required(CONF_PORT): cv.port,
         vol.Optional(CONF_USERNAME): cv.string,
         vol.Optional(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_PROTOCOL): cv.string,
```

Comment:
This should be:

```suggestion
        vol.Optional(CONF_PROTOCOL, default=DEFAULT_PROTOCOL): : vol.In(PROTOCOLS),
```

with `PROTOCOLS` = ['https', 'http'] and `DEFAULT_PROTOCOL = 'https'`.